scipy/stats/_distn_infrastructure.py

```diff
@@ -1821,6 +1829,22 @@ class rv_continuous(rv_generic):
 
     Examples
     --------
+    To demonstrate the difference between theoretical bounds (`a` and `b`)
+    and the adjusted bounds after applying `loc` and `scale` using `support()`:
+
+    >>> from scipy.stats import uniform, beta
+    >>> p = uniform(loc=2, scale=5)
+    >>> print("Theoretical bounds: a =", p.a, ", b =", p.b)
+    Theoretical bounds: a = 0.0 , b = 1.0
+    >>> print("Adjusted bounds with loc and scale:", p.support())
+    Adjusted bounds with loc and scale: (np.float64(2.0), np.float64(7.0))
+
+    >>> p = beta(a=100, b=100, loc=2, scale=5)
+    >>> print("Theoretical bounds: a =", p.a, ", b =", p.b)
+    Theoretical bounds: a = 0.0 , b = 1.0
+    >>> print("Adjusted bounds with loc and scale:", p.support())
+    Adjusted bounds with loc and scale: (np.float64(2.0), np.float64(7.0))
+
```

Comment:
```suggestion
```

This also isn't really the place for an example because it's showing how to create a new distribution, whereas users typically use existing instances (objects), like `scipy.stats.uniform` and `scipy.stats.beta`. So I would remove this.

Also, we probably don't want to demonstrate use of these attributes because they never should have been public as-implemented. Instead, please just add `support` to the list of methods that appear on each distribution's page:

<img width="876" alt="image" src="https://github.com/user-attachments/assets/eefa9563-2c5f-4cf2-81b4-48f50535cecb" />

and maybe find where the Example section is defined and show the use of support. That way it will render as an example for all distributions.

<img width="756" alt="image" src="https://github.com/user-attachments/assets/d9bf254c-db29-4a92-b43b-0d5ab7584fe5" />